### ISSUE TITLE
Dont forget Zones 

### ISSUE THREAD

**Issue by synchromesh62** [TheGameCreators/GameGuruRepo#2485]
This was actually fixed once or has not been re-enabled ?
Zones at one point did ride the terrain so you could adjust them 
Right now you cannot. 

![Zones](https://user-images.githubusercontent.com/35708217/157922388-be485af0-38ec-4e0e-ad19-a5431132b911.png)

**Comment by LeeBamberTGC** [comment]
@synchromesh62 Yes this was redacted as we got an equal number of users complaining the other way. The idea now is we rewrite and improve the zone system to something more intuitive, like painting down an area that becomes the zone, and expand the capabilities of the zone (trigger based on distance, etc).
